promotion/src/main/java/org/cryptimeleon/incentive/promotion/PromotionPoints.java

```diff
@@ -0,0 +1,11 @@
+package org.cryptimeleon.incentive.promotion;
+
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@Value
+@AllArgsConstructor
+public class PromotionPoints {
```

Comment:
I'd also switch this out for potentially different update operations.

Reply:
Which use-cases do you see for that? Currently, we only add k points and then issue a SQSEQ signature to the new token. I do not see how we could profit from having a more general update operation, do you have some ideas?

Reply:
Added more complex update operations, can now be defined by a ProofOfPartialKnowledge